scripts/measures/get-inverse-measures-from-pdfs.js

```diff
@@ -0,0 +1,59 @@
+// running this will print out whether each measure is an inverse measure
```

Comment:
can you update the readme to include instructions for running this script in instructions for how to generate the whole pdf

Reply:
Sorry, what do you mean?

Reply:
When starting at the top of the file I thought that this could be added to steps to generate the measures-data spec. Looks like those instructions don't exist anywhere though.  @marimiyachi or @jiahuang  do you have instructions anywhere for how to recreate measures-data that we could include this step in? I want to make sure that the inverse measures are generated correctly if the data has to be updated for some reason.

Reply:
I follow. I can change the script to output the results to JSON (and keep the resulting file under source control) and then add a step to `scripts/measures/enrich-measures-data.js` which would use that file. Let me do that now, should be quick 👍